fenix/app/src/main/java/org/mozilla/fenix/compose/button/FloatingActionButton.kt

```diff
@@ -35,6 +37,7 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * @param modifier [Modifier] to be applied to the action button.
  * @param contentDescription The content description to describe the icon.
  * @param label Text to be displayed next to the icon.
+ * @param elevation Elevation parameter set.
```

Comment:
```suggestion
 * @param elevation  FloatingActionButtonElevation used to resolve the elevation for this FAB in different states.
```
This is from the `FloatingActionButton` kDoc from for this parameter.